person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accessor :id, :name, :age
+  attr_reader :parent_permission
```

Comment:
[OPTIONAL] You could instead move `:parent_permission` to the `attr_accessor`, as parents will probably give permissions to their children according to the situation.